docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/02_instrument_format/PHP_instrument.md

```diff
@@ -0,0 +1,220 @@
+
+The following configurations should be modified for all instruments.
+
+- In the instrument file name, the keyword `TEMPLATE` should be replaced with the `TEST_NAME` value you chose above.
+- In the instrument class name, the keyword `TEST_NAME` should be replaced with the `TEST_NAME` value.
+- In the `setup()` function, the variable `$this->testName` should be set to the `TEST_NAME` value.
+- In the `setup()` function, the variable `$this->table` should be set to desired name of the SQL table in the database. The table name does not have to be the same as `TEST_NAME` but having be the exact same string is an option and is commonly done.
+- In the `_setupForm()` function the keyword `<TEST_NAME>` in the `preg_match()` function should be replaced by the `TEST_NAME` value.
+- Many other Configuration can alter the behaviour of the instrument and automated quality assurance that is undergoes. Refer to the [Additional Configurations](# Additional Configurations) section below for more information.
+
+Finally, the `getFullName()` function must be changed to reflect the correct full name of the instrument.
+
+#### Pages
+
+Instrument can be broken down into pages. An instrument has a minimum of one page and no maximum number of pages. Pages can have customized names and are configured in the instrument's `getSubtestList()` function. 
+
+The instrument class' `_main()` function represents the first page, also referred to as the "Top" page. All subsequent pages are commonly implemented as `_page1()`, `_page2()`, ... Although this is the most common use case, it is possible to name the functions differently as long as all other configurations are modified accordingly.
+
+In the `getSubtestList()`, make sure to properly refactor the `Name` key of each element in the array to match the following convention `<TEST_NAME><PAGE_FUNCTION_NAME>` where `<Test_NAME>` is replaced by the `TEST_NAME` value you have chosen and `<PAGE_FUNCTION_NAME>` is replaced by `_page1`, `_page2` or the appropriate value if you elected to rename the page functions.
+
+If you chose to rename the page functions (differently than sequential `_pageX`) you must modify the `_setupForm()` function to make sure the `preg_match()` call is properly configured to match all your different pages. This `preg_match()` allows for proper routing to the pages of the instrument.
+
+#### Form Elements
+
+As mentioned above, inheritance from the `NDB_BVL_Instrument.class.inc` enables the instrument under development to use several pre-implemented form wrappers to generate all sorts of fields. The full list of usable wrappers spans across the classes `php/libraries/NDB_BVL_Instrument.class.inc`, `php/libraries/NDB_Page.class.inc` and `php/libraries/LorisForm.class.inc`. 
```

Comment:
LorisForm should not be used directly. Everything should be wrapped in NDB_Page or NDB_BVL_Instrument.. the only instruments directly accessing the LorisForm object should be legacy instruments that were coded before the wrappers were added. I don't think the existence of LorisForm should be advertised here, it just makes it harder to remove